recipes/feather-format/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
+    # Building Feather requires a C++11 compiler, which requires OS X 10.9+
+    export MACOSX_DEPLOYMENT_TARGET=10.9
```

Comment:
https://github.com/wesm/feather/tree/master/python recommends setting to 10.10 actually.